### ISSUE TITLE
Request for K-fold cross-validation support

### ISSUE THREAD

**Issue by ErfanThinker** [songyouwei/ABSA-PyTorch#32]
Is it possible to add K-fold cross-validation support to the code?

**Comment by songyouwei** [comment]
Most ABSA papers do not use K-fold cross-validation in their research, and there is no validation set. I think it would be a good idea to add K-fold cross-validation support in a new branch.

**Comment by ErfanThinker** [comment]
Actually it can be used on bing liu's datasets as used in
https://link.springer.com/article/10.1007/s10586-017-1096-9
(10-fold cross-validation)

**Comment by ThomasK427** [comment]
Are the hyper-parameters finetuned on the test set?

**Comment by songyouwei** [comment]
@ThomasK427 Codes for hyper-parameter tuning are not included in this repo. You can randomly split 20% of training set as the DEVset, and tune hyper-parameters on this DEVset.

**Comment by songyouwei** [comment]
Supported in [this commit](https://github.com/songyouwei/ABSA-PyTorch/commit/db1e350883e0894af4608a7c548f9353c9229b0e).
